### PR TITLE
Examples: Readme Styles

### PR DESCRIPTION
**Pull request by ax3l**
- unify citation style
- unify headline `DirName: Title`
- small style issues

### CODE REVIEW

File: examples/Bremsstrahlung/README.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Emission of Bremsstrahlung from Laser-foil interaction
-======================================================
+Bremsstrahlung: Emission of Bremsstrahlung from Laser-foil interaction
```
Comment (PrometheusPi) [inline examples/Bremsstrahlung/README.rst]: OPTIONAL: 
everywhere you capitalized the title words except here with `interaction`

Comment (ax3l) [inline examples/Bremsstrahlung/README.rst]: Dang!

Comment (ax3l) [inline examples/Bremsstrahlung/README.rst]: `Laser-Foil Interaction` it is

**Comment by ax3l** [comment]
@PrometheusPi updated, thanks!

**Pull request merged**